app/models/certificate_authority.rb

```diff
@@ -1,5 +1,5 @@
 class CertificateAuthority < ApplicationRecord
-  validates :key, presence: true, uniqueness: true, case_sensitive: false,
+  validates :key, presence: true, uniqueness: { case_sensitive: false },
```

Comment:
Was the top-level `case_sensitive` key previously in error? Or is this part of the upgrade?

Reply:
good question 🤔

Reply:
Unsure if it just silently ignored before, but there were errors for "no such class CaseSensitiveValidator" hence me having to make this change